save-common/src/commonMain/kotlin/org/cqfn/save/entities/Result.kt

```diff
@@ -0,0 +1,18 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.domain.ResultStatus
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property id id of test
+ * @property status of test
+ * @property date date of result
+ */
+@Entity
+@Serializable
+data class Result(
+    @Id val id: Int,
+    val status: ResultStatus,
+    val date: String  // fixme should be date
```

Comment:
I would actually use date here (json parsers support it automatically)

Reply:
This is common class now, we could either use `kotlinx.datetime` here or move to `jvmMain` source set and create a dto in common